dwave/experimental/fast_reverse_anneal/api.py

```diff
@@ -39,7 +40,18 @@
 """
 
 
-def get_parameters(sampler: Optional[Union[DWaveSampler, Solver]] = None,
+@cache
+def get_solver_name() -> str:
+    """Find a solver that support fast reverse anneal and return its name.
```

Comment:
```suggestion
    """Find a solver that supports fast reverse anneal and return its name.
```

alternatively "Return the name of a solver that supports fast reverse anneal"